woocommerce/woocommerce-ios
# Dark Mode: update colors for login with updated `WordPressAuthenticator`

Login colors for #1555 

- [x] ⚠️ update PR base to `release/3.2` after https://github.com/woocommerce/woocommerce-ios/pull/1564 is merged ⚠️ 

`WordPressAuthenticator` PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/164

## Changes
- Updated `WordPressAuthenticator` pod to the beta version - I will point it to the official version after the `WordPressAuthenticator` PR is approved and released
- Updated colors we pass to `AuthenticationManager`
- Used primary button style for the Log In button
- Updated colors in `StorePickerViewController`, and always present it fullscreen

## Testing

- Launch the app
- Log out
- Log in --> the colors should look as expected

## Example screenshots

Log in | Log in - highlighted | Log in with email | Log in with email - ready | Request Link | 2FA | store picker
-- | -- | -- | -- | -- | -- | --
![Simulator Screen Shot - iPhone 8 - 2019-12-06 at 10 27 44](https://user-images.githubusercontent.com/1945542/70290547-391f0180-1813-11ea-88a5-e3a92e12f80a.png) | ![Simulator Screen Shot - iPhone 8 - 2019-12-06 at 10 27 45](https://user-images.githubusercontent.com/1945542/70290548-39b79800-1813-11ea-81aa-54112baefda5.png) | ![Simulator Screen Shot - iPhone 8 - 2019-12-06 at 10 30 03](https://user-images.githubusercontent.com/1945542/70291330-d0855400-1815-11ea-90fe-e5584a7b8754.png) | ![Simulator Screen Shot - iPhone 8 - 2019-12-06 at 10 30 11](https://user-images.githubusercontent.com/1945542/70291331-d0855400-1815-11ea-8ae4-6abb95ce940a.png) | ![Simulator Screen Shot - iPhone 8 - 2019-12-06 at 10 39 03](https://user-images.githubusercontent.com/1945542/70291359-e561e780-1815-11ea-9d8c-9e07e88378d7.png) | ![Simulator Screen Shot - iPhone 8 - 2019-12-06 at 10 44 22](https://user-images.githubusercontent.com/1945542/70291360-e5fa7e00-1815-11ea-930f-748d57c9fe03.png) | ![Simulator Screen Shot - iPhone 8 - 2019-12-06 at 10 42 15](https://user-images.githubusercontent.com/1945542/70291262-96b44d80-1815-11ea-96bd-f00b735b5fce.png)
![Simulator Screen Shot - iPhone 8 - 2019-12-06 at 10 27 33](https://user-images.githubusercontent.com/1945542/70290564-463bf080-1813-11ea-8539-0d5b271a5e0a.png) | ![Simulator Screen Shot - iPhone 8 - 2019-12-06 at 10 27 36](https://user-images.githubusercontent.com/1945542/70290565-46d48700-1813-11ea-84ac-a0c1188596e9.png) | ![Simulator Screen Shot - iPhone 8 - 2019-12-06 at 10 30 25](https://user-images.githubusercontent.com/1945542/70291407-0d514b00-1816-11ea-9933-4da8d3fe71b7.png) | ![Simulator Screen Shot - iPhone 8 - 2019-12-06 at 10 30 30](https://user-images.githubusercontent.com/1945542/70291408-0d514b00-1816-11ea-9eef-5e46f262f28d.png) | ![Simulator Screen Shot - iPhone 8 - 2019-12-06 at 10 39 09](https://user-images.githubusercontent.com/1945542/70291409-0d514b00-1816-11ea-96eb-d95521f9e126.png) | ![Simulator Screen Shot - iPhone 8 - 2019-12-06 at 10 44 33](https://user-images.githubusercontent.com/1945542/70291411-0d514b00-1816-11ea-9079-8741236c552c.png) | ![Simulator Screen Shot - iPhone 8 - 2019-12-06 at 10 42 01](https://user-images.githubusercontent.com/1945542/70291279-a0d64c00-1815-11ea-93a9-8cb95ec17327.png)



Update release notes:
